java/code/src/com/redhat/rhn/frontend/xmlrpc/org/OrgHandler.java

```diff
@@ -327,15 +327,15 @@ private ChannelFamily lookupChannelFamily(String channelFamilyLabel) {
      * @return list of systems migrated.
      * @throws FaultException A FaultException is thrown if:
      *   - The user performing the request is not an organization administrator
-     *   - The user performing the request is not a satellite administrator, but the
+     *   - The user performing the request is not a product administrator, but the
```

Comment:
```suggestion
     *   - The user performing the request is not a #product() administrator, but the
```

But I am unsure if this macro works in normal Java

Reply:
I did not use the macro on purpose, see my previous comment for the explanation.